crates/sui-graphql-rpc/src/context_data/module_resolver.rs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
This is also another file that we should have some shared infra on and i believe this already exists? cc @lutianzhou001

Reply:
@bmwill did you mean to tag @longbowlu instead?

Reply:
yes i did :(